containers/utils/openimages/run.bat

```diff
@@ -0,0 +1 @@
+docker run --mount type=bind,source="%cd%",target=/down -i -t down data.json
```

Comment:
Do we need a cross platform version of this?

Reply:
I'm implementing this to be run within the GUI. just wanted to get the CI set up.